extension/capture-options.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: Capture Options
-description: Fiddler Jam extension capturing and recording options
+title: Advanced Options
+description: Fiddler Jam extension advanced capturing and recording options.
 slug: fj-capture-options
 tags: record Fiddler Jam, Fiddler Jam options, Fiddler Jam capturing options, Fiddler Jam recording
 publish: true
 position: 40
 ---
 
 
-# Capture Options
+# Advanced Options
```

Comment:
Calling some options "advanced" is suitable only if this word exists in the tool's interface. Is this the case? Otherwise the word "advanced" has no specific meaning.